src/library/autodj/dlgautodj.cpp

```diff
@@ -372,6 +380,22 @@ void DlgAutoDJ::updateSelectionInfo() {
     }
 }
 
+void DlgAutoDJ::updateInfo() {
+    double duration = 0.0;
+
+    int rows = m_pAutoDJTableModel->rowCount();
+    for (int row = 0; row < rows; ++row) {
+        TrackPointer pTrack = m_pAutoDJTableModel->getTrack(m_pAutoDJTableModel->index(row, 0));
+        duration += pTrack->getDuration();
+    }
+
+    QString label;
+    label.append(mixxx::DurationBase::formatTime(duration));
+    label.append(QString(" (%1)").arg(rows));
```

Comment:
here we should reuse the https://github.com/mixxxdj/mixxx/blob/f2d670edeb483efb066bf7f00255d01df2f0017c/src/library/trackset/playlistfeature.cpp#L26
in fact the playlists share the same issue that it shows the max time only and not the resulting time when put to Auto-DJ.